### PR TITLE
Release 20.05

### PR DESCRIPTION
**Pull request by Denis-Averin**
- Switched to API v3.0
- Tests and documentation added

### CODE REVIEW

File: src/Model/GeneratorParamsList.cs [hunk 1]
```diff
@@ -23,48 +23,51 @@
...
+    public class GeneratorParamsList
+    {
         /// <summary>
-        /// List of barcode builders
-        /// </summary>  
-        public List<BarCodeBuilder> BarCodeBuilders { get; set; }
-		
+        ///     List of barcode generators             
+        /// </summary>
+        public List<GeneratorParams> BarcodeBuilders { get; set; }
```
Comment (eldar-yakupov) [inline src/Model/GeneratorParamsList.cs:45]: Shouldn't we remove all the mentions of Builder from the project?

File: docs/GeneratorParamsList.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Aspose.BarCode.Cloud.Sdk.Model.GeneratorParamsList
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**BarcodeBuilders** | [**List&lt;GeneratorParams&gt;**](GeneratorParams.md) | List of barcode generators              | [optional]
```
Comment (eldar-yakupov) [inline docs/GeneratorParamsList.md:6]: Builder

File: src/Model/GeneratorParamsList.cs (hunk 1 above)
Comment (denex) [reply src/Model/GeneratorParamsList.cs:45]: > Shouldn't we remove all the mentions of Builder from the project?

I suppose we keep it in order of backward compatibility.

File: docs/GeneratorParamsList.md (hunk 2 above)
Comment (denex) [reply docs/GeneratorParamsList.md:6]: Removing *builder* will break backward compatibility

**Pull request merged**